src/Workflow/TransitionAutomater.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\Workflow;
+
+use Symfony\Component\Workflow\Event\Event;
+use Symfony\Component\Workflow\Transition;
+use function array_map;
+
+final class TransitionAutomater
+{
+    public function onEntered(Event $event) : void
+    {
+        $subject = $event->getSubject();
+        $workflow = $event->getWorkflow();
+
+        $transitions = array_map(
+            function (Transition $transition) : string {
+                return $transition->getName();
+            },
+            $workflow->getEnabledTransitions($subject)
+        );
+
+        foreach ($transitions as $transition) {
+            if (!$workflow->can($subject, $transition)) {
+                continue;
+            }
+
+            $workflow->apply($subject, $transition);
```

Comment:
There's probably a better way to do this.

Reply:
Why does it `break`? Using only one of the multiple transitions?

Reply:
`return` is probably clearer. Finds the next transition that can be applied.